### PR TITLE
Upgrade testcafe version

### PR DESCRIPTION
**Pull request by tianfeng92**
**Thank you for contributing to the `sauce-docs` project!**
**A well described pull request helps maintainers quickly review and merge your change**

Before submitting your PR, please check our [contributing](https://github.com/saucelabs/sauce-docs/blob/master/CONTRIBUTING.MD)
guidelines, applied for this repository. Avoid large PRs, help reviewers by making them as simple 
and short as possible.

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [ ] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/master/CONTRIBUTING.MD) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/testrunner-toolkit.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ Each docker image tag is the 'latest' image that supports the specific framework
 
 | TestCafe Version   | Supported Browsers        | Release Notes                                                                                                 | Sauce Cloud Support [[?](testrunner-toolkit/running-tests.md#test-on-sauce-labs)] |
 |--------------------|---------------------------|---------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------|
-| 1.8.5              | Please see release notes. | [saucelabs/sauce-testcafe-runner](https://github.com/saucelabs/sauce-testcafe-runner/releases/tag/v0.1.16)    |                                                                                   |
+| 1.11.0              | Please see release notes. | [saucelabs/sauce-testcafe-runner](https://github.com/saucelabs/sauce-testcafe-runner/releases/tag/v0.3.0)    |                                                                                   |
```
Comment (alexplischke) [inline docs/testrunner-toolkit.md:70]: Why replace the entry and not keep both?

Comment (dpgraham) [reply docs/testrunner-toolkit.md:70]: ^ Agreed, and have the newer versions above the older versions.

Comment (tianfeng92) [reply docs/testrunner-toolkit.md:70]: ok.

File: docs/testrunner-toolkit.md [hunk 2]
```diff
@@ -68,6 +68,7 @@ Each docker image tag is the 'latest' image that supports the specific framework
 | TestCafe Version   | Supported Browsers        | Release Notes                                                                                                 | Sauce Cloud Support [[?](testrunner-toolkit/running-tests.md#test-on-sauce-labs)] |
 |--------------------|---------------------------|---------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------|
 | 1.8.5              | Please see release notes. | [saucelabs/sauce-testcafe-runner](https://github.com/saucelabs/sauce-testcafe-runner/releases/tag/v0.1.16)    |                                                                                   |
+| 1.11.0              | Please see release notes. | [saucelabs/sauce-testcafe-runner](https://github.com/saucelabs/sauce-testcafe-runner/releases/tag/v0.3.0)    |                                                                                   |
```
Comment (alexplischke) [inline docs/testrunner-toolkit.md:71]: That entry should be at the top, not bottom :)

**Review by alexplischke — CHANGES_REQUESTED** [review]

File: docs/testrunner-toolkit.md (hunk 2 above)
Comment (tianfeng92) [reply docs/testrunner-toolkit.md:71]: oh I see..

**Review by spider-sauce — APPROVED** [review]

**Review by alexplischke — APPROVED** [review]

**Pull request merged**